src/ContaoCommunityAlliance/Contao/Bindings/Subscribers/NewsSubscriber.php

```diff
@@ -103,13 +103,25 @@ public function handleNews(GetNewsEvent $event, $eventName, EventDispatcherInter
 
         // Clean the RTE output.
         if ($newsModel->teaser != '') {
-            if ($objPage->outputFormat == 'xhtml') {
-                $objTemplate->teaser = \String::toXhtml($newsModel->teaser);
+            // PHP 7 compatibility
+            // See #309 (https://github.com/contao/core-bundle/issues/309)
+            if (version_compare('3.5.5', VERSION, '>=')) {
```

Comment:
Should be:
```diff
- if (version_compare('3.5.5', VERSION, '>=')) {
+ if (version_compare('3.5.5', VERSION . '.' . BUILD, '>=')) {
```
Otherwise the build is not contained.